consensus/src/pending_votes.rs

```diff
@@ -211,6 +228,42 @@ impl PendingVotes {
 
         VoteReceptionResult::VoteAdded(voting_power)
     }
+
+    pub fn log_voting_power_if_proposer(&self) {
+        // If we don't have any votes, we are not the proposer
+        if self.li_digest_to_votes.is_empty() {
```

Comment:
This is misleading as you might have no votes even if you are a proposer and we want to log 0 in that case. You can pass is_proposer as a flag in `process_certificates` function to deal with this. Also, since all of this are gauges, you need to reset the gauges for non proposers. Ideally for non-proposers, you can just set the gauge value to -1, so that we can filter -1 values in dashboard to get the data we are looking for.

Reply:
we are sampling the metrics when showing in grafana, so having -1 set will make it such that rounds when we do have values will not show up on the dashboard unless we got lucky and metric collection happen in between when we were a proposer and when we were not.

these are basically showing value from last round we were proposing, which for dashboard seems fine?

we can also log an info here, so that we can actually see/correlate in humio only instances where you actually were a proposer?